packages/react-paypal-js/.eslintrc.json

```diff
@@ -27,6 +27,7 @@
         }
     },
     "rules": {
+        "jsdoc/no-undefined-types": 1,
```

Comment:
This allows importing types only for use in `jsdoc` without the linter reporting an unused variable error.